ppr-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ppr-ui",
-  "version": "0.3.53",
+  "version": "0.3.54",
```

Comment:
Thanks for the update. **package-lock.json** should have an updated version as well. Maybe forgot to run `npm install` ?

Reply:
Done